Makefile

```diff
@@ -278,7 +278,8 @@ install: build
 	${INSTALL} ${PROTOC_GEN_SWIFT} ${BINDIR}
 
 clean:
-	swift build --clean
+	-swift build --clean
```

Comment:
Tells make to ignore these lines if they fail.
So whichever one is not supported on the local platform will not stop the build.  (I considered just dropping this part, since `rm -rf .build` does the same thing.)